tests/psmdbcluster/psmdb_cluster_test.go

```diff
@@ -85,6 +88,36 @@ func TestPSMDBClusterAPI(t *testing.T) {
 	}
 	assert.True(t, clusterFound)
 
+	t.Log("Wating for cluster to be ready")
+	err = waitForPSMDBClusterState(tests.Context, kubeconfig, name, controllerv1beta1.PSMDBClusterState_PSMDB_CLUSTER_STATE_READY)
+	require.NoError(t, err)
+
+	updateReq := &controllerv1beta1.UpdatePSMDBClusterRequest{
+		KubeAuth: &controllerv1beta1.KubeAuth{
+			Kubeconfig: kubeconfig,
+		},
+		Name: name,
+		Params: &controllerv1beta1.PSMDBClusterParams{
+			ClusterSize: 6,
+			Replicaset: &controllerv1beta1.PSMDBClusterParams_ReplicaSet{
+				ComputeResources: &controllerv1beta1.ComputeResources{},
+				DiskSize:         1024 * 1024 * 1024,
+			},
+		},
+	}
+
+	t.Log("First update")
+	upresp, err := tests.PSMDBClusterAPIClient.UpdatePSMDBCluster(tests.Context, updateReq)
+	assert.NoError(t, err)
+	assert.NotNil(t, upresp)
```

Comment:
can we check that values are update?

Reply:
I modified the test but there is not much I can do before running out of resources
```
Events:
  Type     Reason            Age                 From               Message
  ----     ------            ----                ----               -------
  Warning  FailedScheduling  33s (x4 over 108s)  default-scheduler  0/1 nodes are available: 1 Insufficient cpu.
```

Reply:
yeah, but even with that we should be able get specs for the cluster.